lib/govuk_message_queue_consumer/consumer.rb

```diff
@@ -11,7 +11,12 @@ class Consumer
     NUMBER_OF_MESSAGES_TO_PREFETCH = 1
 
     def self.default_connection_from_env
-      Bunny.new(GovukMessageQueueConsumer::RabbitMQConfig.from_environment(ENV))
+      # https://github.com/ruby-amqp/bunny/blob/066496d/docs/guides/connecting.md#paas-environments
+      if ENV["RABBITMQ_URL"]
```

Comment:
Yeah, it's a fair point. There's a tradeoff between readability and covering that edge case. I think in this case I'd go with the more readable option because we'll be getting rid of the old env vars within a matter of months (if not sooner) and at that point, RABBITMQ_URL will become mandatory anyway.

Reply:
fair enough :+1:

Reply:
Oh, it looks like we're planning on [setting it to the empty string](https://github.com/alphagov/govuk-puppet/commit/64636ee). Let's go with your suggestion and save faffing with the Puppet code :)